electron/renderer/src/components/LoadingSpinner.css

```diff
@@ -0,0 +1,33 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+#loading-spinner-wrapper {
+  display: flex;
+  height: 100vh;
+  position: absolute;
+  width: 100%;
+  z-index: 99;
+  transition: opacity 0.25s ease-in-out;
+  -moz-transition: opacity 0.25s ease-in-out;
+  -webkit-transition: opacity 0.25s ease-in-out;
```

Comment:
does not compiler add these automatically?

Reply:
What compiler? I am using css :)

Reply:
Bardia means something like postCSS as a CSS pre-compiler.

Reply:
Ah okay, we don't have that in wire-desktop.